linera-execution/tests/wasm.rs

```diff
@@ -25,6 +26,7 @@ use test_case::test_case;
 /// called correctly and consume the expected amount of fuel.
 ///
 /// To update the bytecode files, run `linera-execution/update_wasm_fixtures.sh`.
+#[ignore]
```

Comment:
(This is just temporary, is it? Because you haven't updated the numbers yet?)

Reply:
Yes, temporary.

Reply:
Addressed.